src/main/java/com/github/underscore/lodash/Xml.java

```diff
@@ -960,18 +960,22 @@ private static XmlStringBuilder checkStandalone(String encoding, XmlStringBuilde
     private static String getRootName(final Map localMap) {
         int foundAttrs = 0;
         int foundElements = 0;
+        int foundListElements = 0;
         if (localMap != null) {
             for (Map.Entry entry : (Set<Map.Entry>) localMap.entrySet()) {
                 if (String.valueOf(entry.getKey()).startsWith("-")) {
                     foundAttrs += 1;
                 } else if (!String.valueOf(entry.getKey()).startsWith(COMMENT)
-                        && !String.valueOf(entry.getKey()).startsWith("?")
-                        && (!(entry.getValue() instanceof List) || ((List) entry.getValue()).size() <= 1)) {
+                        && !String.valueOf(entry.getKey()).startsWith("?")) {
+                    if (entry.getValue() instanceof List
+                     && ((List) entry.getValue()).size() > 1) {
```

Comment:
It may be in the same line with previous condition.

Reply:
Done.